### DISCUSSION TITLE
What is allorf in the antismash prediction?

### DISCUSSION THREAD

**Discussion by Subhajeet1997** [antismash/antismash#415] [Q&A]
I was predicting BCGs of cyanobacteria using Antismash. In the predicted result, when I was looking into the predicted lanthipeptide.
I got two kinds of core peptides one is normally such as "NKJHNKOJ_00532 leader/core peptide" another one is "allorf_0641468_0641545 leader/core peptide". In the allorf_0641468_0641545 leader/core peptide, the core peptide is of short stretch and seems incomplete. I want to know what is the meaning of allorf. I have searched but can't get any info regarding this.

**Comment by SJShaw** [comment]
The `allorf` features are those found by antiSMASH doing some gene finding to find any precursors that most gene finding tool configurations exclude because they're so small.

If you're happy that your initial gene annotations are completely correct, you can ignore that additional result.
